src/pages/index.js

```diff
@@ -105,36 +138,75 @@ function handleAddNewCardButton() {
 }
 
 function handleProfileFormSubmit(inputValues) {
+  document.querySelector(settings.submitButtonSelector).textContent =
```

Comment:
`document.querySelector(settings.submitButtonSelector)`


Please, don't search the buttons in each line where you need them. 


It’s better to find **constants** only 1 time at the top of the file not to waste resources on searching them again and again when you call a method (function)